[UI]: style fix of few components in dark theme

samyakshah3008 opened the pull request:
**Notes for Reviewers**

- This PR aims to fix dark theme report  #7863 number 2, 4 and modal close button add. 
- I'll be raising one separate issue ticket to fix number 1 and 3 of issue ticket #7863. 

## Checklist of expected changes: 

- [X] Radio buttons style change to action color mentioned in figma file (Refer number 2 of issue ticket)
- [X] Switch Component style is according to action color (Refer number 4 of issue ticket)
- [X]  Add close button and title in modal  (Refer the modal proposal section of issue ticket) 

## Before Change Screenshot: 

![image](https://github.com/meshery/meshery/assets/86982322/bda1e51f-b0ed-4503-ba1c-760a94081f32)

![image](https://github.com/meshery/meshery/assets/86982322/7a179ac4-fddd-46ac-8b25-8c15f62666f8)

## After Change Screenshot: 

![92951825-ad78-436c-a7f2-a856374e97df-min](https://github.com/meshery/meshery/assets/86982322/f84aa3ea-83be-43e5-b134-70e3aa427619)

![image](https://github.com/meshery/meshery/assets/86982322/52239a7b-e810-4882-a6f2-7da3eb94c403)

This PR fixes #

#7863 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

vishalvivekm commented:
@samyakshah3008 
Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

Yashsharma1911 commented on ui/components/MesherySettingsPerformanceComponent.js:
```diff
@@ -41,6 +41,11 @@ const styles = (theme) => ({
   },
+  radio: {
+    '&.Mui-checked': {
+      color: "#00b39f!important",
```
same here

samyakshah3008 replied:
Sure, thanks!

Yashsharma1911 commented on ui/components/MesheryPerformance/PerformanceProfiles.js:
```diff
@@ -311,7 +311,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
           Content={
+            <>
             <Paper className={classes.addProfileModal} >
+            <span onClick={()=> setProfileForModal(undefined)}>Close</span>
```
Don't add just close text, take reference from other components you can add cross icon over here

Yashsharma1911 replied:
ohh you already mentioned in the above checklist 👍

samyakshah3008 replied:
Yeah 😄

samyakshah3008 commented:
Thanks for all your feedbacks  @Yashsharma1911  , as PR is already draft, I'll make the requested changes + will complete the pending work before marking the PR ready for review.
One blocker which I would like to point out is:
Code changes are quite less but as I am newly contributing to this repo, I am also investing some time understanding the codebase.
Tentative deadline to mark this PR ready for review:
By today, 21st June.
Cheers!

samyakshah3008 commented on ui/components/MesheryPerformance/PerformanceProfiles.js:
```diff
@@ -311,7 +311,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
         <GenericModal
           open={!!profileForModal}
           Content={
+            <>
```
Resolved, removed ✅
The only reason I added a fragment because I was thinking to add another parent for cross button but later I did it with second approach.

samyakshah3008 commented on ui/components/MesheryPerformance/index.js:
```diff
@@ -163,6 +163,11 @@ const styles = (theme) => ({
   },
+  radio: {
+    '&.Mui-checked': {
+      color: "#00b39f!important",
```
Sure, pushing my latest changes in few moments.

samyakshah3008 commented:
## Latest change: 

- Completed milestone 3rd, fixed modal with fully functional close button with proper css alignments for both dark and light themes. 
- Addressed @Yashsharma1911 review feedback comments. 

### Dark theme: 

![image](https://github.com/meshery/meshery/assets/86982322/52239a7b-e810-4882-a6f2-7da3eb94c403)

### Light theme: 

![image](https://github.com/meshery/meshery/assets/86982322/af0916bf-1541-4ba8-86d9-3b7aaf34be08)

samyakshah3008 commented:
@Yashsharma1911 can you please re-review?

Freedisch commented:
@samyakshah3008 Make sure to always fix any Eslint issue locally before u commit

vaibhav135 commented on ui/components/MesheryPerformance/PerformanceProfiles.js:
```diff
@@ -311,38 +322,45 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
-                runTestOnMount={!!profileForModal?.runTest}
-              />
-            </Paper>
+                  loadAsPerformanceProfile
+                  // @ts-ignore
+                  performanceProfileID={profileForModal?.id}
+                  // @ts-ignore
+                  profileName={profileForModal?.name}
+                  // @ts-ignore
+                  meshName={profileForModal?.service_mesh}
+                  // @ts-ignore
+                  url={profileForModal?.endpoints?.[0]}
+                  // @ts-ignore
+                  qps={profileForModal?.qps}
+                  // @ts-ignore
+                  loadGenerator={profileForModal?.load_generators?.[0]}
+                  // @ts-ignore
+                  t={profileForModal?.duration}
+                  // @ts-ignore
+                  c={profileForModal?.concurrent_request}
+                  // @ts-ignore
+                  reqBody={profileForModal?.request_body}
+                  // @ts-ignore
+                  headers={profileForModal?.request_headers}
+                  // @ts-ignore
+                  cookies={profileForModal?.request_cookies}
+                  // @ts-ignore
+                  contentType={profileForModal?.content_type}
+                  // @ts-ignore
+                  runTestOnMount={!!profileForModal?.runTest}
+                />
```
Do we still need `// @ts-ignore` after `// @ts-nocheck` at the top of the file. Since `// @ts-nocheck` ignore the whole file.

vaibhav135 reviewed (approved):
LGTM🚀

Yashsharma1911 reviewed (commented):
@samyakshah3008 PR consists eslint issues

Yashsharma1911 reviewed (approved):
// @Chadha93

leecalcote commented on ui/components/MesheryPerformance/PerformanceProfiles.js:
```diff
@@ -16,14 +16,26 @@ import { updateProgress } from "../../lib/store";
+    flexGrow : 1
+  },
+  dialogHeader : {
+    backgroundColor : theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#396679',
```
Are we not able to reference a theme color for `'#396679',`?

samyakshah3008 replied:
Sure thing Lee, 
I have address your change in my latest commit ✅

samyakshah3008 commented on ui/components/MesheryPerformance/index.js:
```diff
@@ -163,6 +162,11 @@ const styles = (theme) => ({
   },
+  radio : {
+    '&.Mui-checked' : {
+      color : theme.palette.type === 'dark' ? "#00B39F" : theme.palette.primary
```
Sure thing Lee, 
I have address your change in my latest commit ✅

samyakshah3008 commented:
## Recent update: 

Made `wrk2` radio button disabled as change requested by Lee. 

### Screenshots: 

In User preferences: 

![image](https://github.com/meshery/meshery/assets/86982322/c7646da9-ccaa-43c5-b3a3-5d364703634f)

In Performance Profile Wizard modal: 

![image](https://github.com/meshery/meshery/assets/86982322/12c6b80c-72b0-4c77-8467-580d76bad3f9)
